frontend/interfaces/host.ts

```diff
@@ -203,8 +203,11 @@ export interface IHost {
   users: IHostUser[];
   device_users?: IDeviceUser[];
   munki?: IMunkiData;
-  mdm_enrollment_status: MdmStatus;
-  mdm_server_url: string;
+  // TODO: Below fields are intentionally redundant
+  // API currently not nested, but will be reverted to nested structure
+  mdm?: IHostMdmData;
+  mdm_enrollment_status?: "On (automatic)" | "On (manual)" | "Pending" | "Off";
+  mdm_server_url?: string;
```

Comment:
Looks like this will be done today if you want to hold: #9505

Reply:
thanks for the heads up